src/api/spec/features/webui/search_spec.rb

```diff
@@ -1,223 +1,226 @@
-require 'sphinx_helper'
+require 'browser_helper'
```

Comment:
But why? It was nicely isolated in that helper. Now you have `thinking_sphinx_test_init` all over the place.

Reply:
They way RSpec helper methods are isolated is [documented](https://relishapp.com/rspec/rspec-core/docs/helper-methods/define-helper-methods-in-a-module#include-a-module-in-only-some-example-groups).

I have already checked, and calling `thinking_sphinx_test_init` in a test that doesn't have `thinking_sphinx: true` results in an `undefined local variable or method 'thinking_sphinx_test_init'` error.

In other words, `thinking_sphinx_test_init` is only available in tests with `thinking_sphinx: true`.

Reply:
The code to init and stop `ThinkingSphinx::Test` was moved from the tests to the `spec/support/thinking_sphinx.rb` file, using [hooks filtered by symbols](https://relishapp.com/rspec/rspec-core/v/3-9/docs/hooks/filters#filtering-hooks-using-symbols).